aws/scripts/create_links_sinks.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+import argparse
+import boto3
+from jinja2 import Environment, FileSystemLoader, select_autoescape
+
+# Set up parser to accept CLI flags
+parser = argparse.ArgumentParser(add_help=True)
+parser.add_argument('--organization', help="AWS Organization", type=str)
+parser.add_argument('--organization-unit', help="AWS Organization Unit", type=str)
+parser.add_argument('--sink-name', help="Human-friendly AWS Metric Sink Name, defaults to \"Centralized-Monitoring\"", type=str)
+parser.add_argument('--profile', help="AWS CLI Config Profile Name for the Management Account, uses your default AWS profile if not specified", type=str)
+parser.add_argument('--regions', help="Double quoted, comma separated list of regions to create a sink and StackSet in. If not specified, ", type=str)
+parser.add_argument('--excluded-accounts', help="[Optional] double quoted, comma separated list of accounts to exclude (ex. --exclude \"12345678910, 09876543210\"). Must exclude suspended accounts", type=str)
+
+# AWS Default regions
+aws_default_regions = ["us-east-2", "us-east-1", "us-west-1", "us-west-2",
+		"ap-south-1", "ap-northeast-3", "ap-northeast-2", "ap-southeast-1", "ap-southeast-2",
+		"ap-northeast-1", "ca-central-1", "eu-central-1", "eu-west-1", "eu-west-2", "eu-west-3",
+		"eu-north-1", "sa-east-1"]
+
+policy = "{\"Version\":\"2012-10-17\",\"Statement\":[{\"Effect\":\"Allow\",\"Principal\":\"*\",\"Action\":[\"oam:CreateLink\",\"oam:UpdateLink\"],\"Resource\":\"*\",\"Condition\":{\"ForAllValues:StringEquals\":{\"oam:ResourceTypes\":\"AWS::CloudWatch::Metric\"},\"ForAnyValue:StringEquals\":{\"aws:PrincipalOrgID\":\"%s\"}}}]}"
```

Comment:
To avoid intense escaping, considering using a docstring `"""` or single quotes.

Reply:
This actually is a constraint applied from the AWS API [docs](https://docs.aws.amazon.com/OAM/latest/APIReference/API_PutSinkPolicy.html)
```
[Policy](https://docs.aws.amazon.com/OAM/latest/APIReference/API_PutSinkPolicy.html#API_PutSinkPolicy_RequestSyntax)
The JSON policy to use. If you are updating an existing policy, the entire existing policy is replaced by what you specify here.

The policy must be in JSON string format with quotation marks escaped and no newlines.
```

Reply:
ah, yep. Replaced to use triple quoted strings and it worked!